### PR TITLE
kubevpn-2.2.19

### PR DESCRIPTION
**Pull request by wencaiwulue**
update kubevpn to 2.2.19, because i recall 2.2.19 and released it again, but BrewTestBot seems will not check it again, but previous pull request has been closed, https://github.com/Homebrew/homebrew-core/pull/193362, so create a new pull request to update version to 2.2.19 manually

### CODE REVIEW

File: Formula/k/kubevpn.rb [hunk 1]
```diff
@@ -1,21 +1,10 @@
 class Kubevpn < Formula
   desc "Offers a Cloud-Native Dev Environment that connects to your K8s cluster network"
   homepage "https://www.kubevpn.cn"
-  url "https://github.com/kubenetworks/kubevpn/archive/refs/tags/v2.2.18.tar.gz"
-  sha256 "bb5c2f19d5dc256794d118a36d9ff2ac7382c38fb7b8ac376e037ea50d99a7c0"
+  url "https://github.com/kubenetworks/kubevpn/archive/refs/tags/v2.2.19.tar.gz"
+  sha256 "4a37db518c9cb79b824addc8852f6a054394c4ca01cea6b4c22722704800bfae"
   license "MIT"
 
-  bottle do
```
Comment (chenrui333) [inline Formula/k/kubevpn.rb:8]: You can keep the bottle block, it would be managed by the bot

Comment (wencaiwulue) [reply Formula/k/kubevpn.rb:8]: ok, got it

Comment (chenrui333) [reply Formula/k/kubevpn.rb:8]: can you revert the change for that part? thanks!

**Review by chenrui333 — APPROVED** [review]

File: Formula/k/kubevpn.rb (hunk 1 above)
Comment (wencaiwulue) [reply Formula/k/kubevpn.rb:8]: yes, i reverted the change of this part, you can review it now 👍

**Comment by wencaiwulue** [comment]
@chenrui333 i rebased the master branch and pushed again, please take a look, thanks ~

**Review by chenrui333 — APPROVED** [review]

**Pull request merged**